project.clj

```diff
@@ -6,13 +6,13 @@
 
   :source-paths ["src/cljs"]
 
-  :dependencies [[org.clojure/clojure "1.7.0"]
+  :dependencies [[org.clojure/clojure "1.9.0"]
 
                  ;; transitive dependencies
-                 [org.clojure/clojurescript "1.7.228"]
+                 [org.clojure/clojurescript "1.10.238" :exclusions [org.clojure/clojure com.fasterxml.jackson.core/jackson-core]]
```

Comment:
Few things:
  * Long lines should be broken up. With at the artifact name one per line here, potentially the `:exclusions` keyword starting a newline (eg https://github.com/puppetlabs/puppetserver/blob/master/project.clj#L174-L178).
  * We tend to prefer explicit conflict resolution of transitive dependencies over exclusions, that's not always possible, but we should probably do that with jackson at least.
 * Looking at these dependencies lists it might be worth while to pull in [clj-parent](https://github.com/puppetlabs/clj-parent) (see puppetserver's usage of clj-parent for an example, reading lein's [docs](https://github.com/technomancy/leiningen/blob/master/doc/MANAGED_DEPS.md) is probably a good first step).